odoo/addons/base/views/ir_module_views.xml

```diff
@@ -174,20 +174,15 @@
                   <field name="website"/>
                   <field name="application"/>
                   <templates>
+                    <t t-name="kanban-menu">
```

Comment:
does it work if we set `groups="...."` on this node ? E.g. only the admin can see the dropdown

Reply:
it should work yes, but I can check

Reply:
yes group works